core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -553,23 +553,26 @@ class TaskManager(taskManagerResources: SystemResources = TaskManagerDefaults.de
       allDone = graphNodesInStatesFor(List(ORPHAN, COMPLETED)).size == graphNodes.size
 
       if (!allDone && runningTasks.isEmpty && tasksToSchedule.isEmpty) {
-        logger.error(s"There are ${readyTasks.size} tasks ready to be scheduled but not enough system resources available.")
-        readyTasks.foreach { readyTask =>
-          val resourcesType: String = readyTask match {
-            case _: FixedResources    => "FixedResources"
-            case _: VariableResources => "VariableResources"
-            case _: Schedulable       => "Schedulable"
-            case _                    => "Unknown Type"
+        if (readyTasks.nonEmpty) {
```

Comment:
I think we check for cyclical dependencies so that's not a state they are in.  Perhaps if they depend on a task in the system?  I don't think an added message is critical, other then perhaps: "Unknown error" or something, just as a marker until we refactor the task system.